libwallet/src/api_impl/owner.rs

```diff
@@ -667,6 +679,19 @@ where
 	)?;
 
 	let keychain = w.keychain(keychain_mask)?;
+
+	// Add our contribution to the offset
+	if context_res.is_ok() {
+		// Self sending: don't correct for in- and outputs here
+		//  as we will do it during finalization.
```

Comment:
Nit: Extra whitespace.

Reply:
This is on purpose, because my IDE detects multi line sentences without an extra space in the beginning as a separate comment. 
This is how it highlights a TODO comment with and without an extra space:
![IDE screenshot](https://i.imgur.com/AI4cGmZ.png)
But I'll change it anyway, since I don't think we do that anywhere else in the codebase.